spec/support/match_contents_matcher.rb

```diff
@@ -1,7 +1,18 @@
 RSpec::Matchers.define :match_contents do |regexp|
   match do |filename|
-    contents = TestPaths.app_path.join(filename).read
-    contents =~ regexp
+    @filename = filename
+    @contents = TestPaths.app_path.join(filename).read
```

Comment:
I like that! The only (minor) drawback is that on test failure it would run the helper twice, and we'd have to pass it the `filename` param. Otherwise, we could memoize it and that would not be an issue.

But as I said, I think that's a very minor issue on test environment and I even prefer the version with the helper method, so I'm doing it. Thanks!